src/main/java/com/google/sps/servlets/utils/Information.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets.utils;
+
+import com.google.api.services.youtube.model.Video;
+import com.google.api.services.youtube.model.VideoListResponse;
+import com.google.api.services.youtube.model.VideoSnippet;
+import com.google.api.services.youtube.model.VideoStatistics;
+
+public class Information {
+  String videoName;
+  String videoAuthor;
+  int numLikes;
+  int numDislikes;
+  String publishDate;
+
+  public Information(VideoListResponse videoResponse) {
+    Video video = videoResponse.getItems().get(0);
+    VideoSnippet videoSnippet = video.getSnippet();
+    VideoStatistics videoStats = video.getStatistics();
+    this.videoName = videoSnippet.getTitle();
+    this.videoAuthor = videoSnippet.getChannelTitle();
+    this.publishDate = videoSnippet.getPublishedAt().toStringRfc3339();
```

Comment:
Question: where is toStringRfc3339 from? Is that a Youtube api function?

Reply:
Yes. This will help display date in a easily readable formate:)